core/controllers/admin.py

```diff
@@ -203,6 +208,26 @@ def _reload_collection(self, collection_id):
                 feconf.SYSTEM_COMMITTER_ID, unicode(collection_id))
         else:
             raise Exception('Cannot reload a collection in production.')
+    def _reload_dummy_exploration(self, dummy_exp_count):
```

Comment:
I don't see any tests for reload demo explorations in `admin_test.py`. Should I write tests for those too? I also see some protractor test for button. And I would be comfortable writing all tests in seperate PR?